bedrock/firefox/templates/firefox/new.html

```diff
@@ -123,15 +129,21 @@ <h1 class="large">
           </div>
           <div class="ios-links-wrapper latest-links-wrapper">
             <ul>
-              {% if l10n_has_tag('mobile_links') %}
-              <li>
-                <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">{{_('Get Firefox on your Android device')}}</a>
-              </li>
-              <li>
-                <a href="{{ url('firefox.os.index') }}">{{_('Learn about Firefox OS')}}</a>
-              </li>
+              {% if l10n_has_tag('ios_announce') %}
+                <li><a href="{{ url('newsletter.ios') }}">{{_('Learn more about Firefox on iOS')}}</a></li>
+              {% else %}
+                {% if l10n_has_tag('mobile_links') %}
```

Comment:
This could be combined into <code>{% elif l10n_has_tag('mobile_links') %}</code> instead of an if inside an else. 
And it seems there's no copy for when neither tag is active, so the space is just blank? Correct?